### DISCUSSION TITLE
Does MoreMcmeta work with EMF and ETF?

### DISCUSSION THREAD

**Discussion by FashionNugget** [MoreMcmeta/core#74] [Core Dev Questions]
Not sure if they function the same way, don't want to have multiple mods doing the same thing.

**Comment by soir20** [comment]
Sorry for the delayed reply; I've had a lot of work on my plate recently. MoreMcmeta core should be compatible with ETF and EMF. MoreMcmeta's handling of textures shouldn't be interfering with other mods' handling of entity models.

ETF adds emissive textures for entities similar to the MoreMcmeta emissive plugin, though MoreMcmeta's plugin additionally supports blocks. I'm not sure what happens if you use both ETF and MoreMcmeta's emissive plugin with an emissive resource pack; the overlay might just be rendered twice for entities.
